docs/misc/faq.md

```diff
@@ -0,0 +1,110 @@
+---
+title: FAQ
+---
+
+Other questions? Ask on `#wg-developer-stonesoup`.
+
+### Q: How do I deliver K8s resources in stages? For example, installing a Custom Resource Definition (CRD) first, then installing the Custom Resource (CR) for that CRD.
+
+As long as your resources are declaratively defined, they will eventually be reconciled with the cluster (it just may take Argo CD a few retries). For example, the CRs might fail before the CRDs are applied, but on retry the CRDs will now exist (as they were applied during the previous retry). So now those CRs can progress.
+
+_However_, this is not true if you are installing an Operator (e.g. Tekton) via OLM `Subscription`, and then using an operand of that operator (e.g. `Pipeline` CRs), at the same time. In this case, you will need to add the `argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true` annotation to the operands (or add it to the parent `kustomization.yaml`).
+
+See the FAQ question '_the server could not find the requested resource_' below for details.
+
+For finer-grained control of resource apply ordering, use Argo CD [Sync waves](https://argoproj.github.io/argo-cd/user-guide/sync-waves/) (Here is an [example](https://github.com/argoproj/argocd-example-apps/tree/master/sync-waves)).
+
+### Q: What if I want my service's K8s resources in a separate Git repository? (i.e. not this one)
+
+Ultimately, as a team, we should decide on a resource deployment strategy going forward: whether we want every team's K8s resources to be defined within this repository (as a GitOps monorepo), or within individual team's Git repositories. IMHO it is easiest to coordinate deployments within a single Git repository (such as this one), rather than multiple independent repositories.
```

Comment:
Is this _your_ opinion? I don't think that FAQs should be written by a single individual, but by the community. Maybe we can restructure this answer to something like (not as a list, of course -- I just wanted to highlight the points)

* Describe how it is done
* BUT this is not the convention that we follow BECAUSE...

Reply:
done